### PR TITLE
Updating the SDK to an SDK that includes globing for source and resources

### PR DESCRIPTION
**Pull request by livarcocc**
Updating the SDK to an SDK that includes globing for source and resources implicitly.

@piotroko @piotrpMSFT @jonsequitur @eerhardt @krwq @jgoshi

### CODE REVIEW

File: build/Microsoft.DotNet.Cli.DependencyVersions.props [hunk 1]
```diff
@@ -2,8 +2,8 @@
 <Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <CLI_MSBuild_Version>15.1.0-preview-000458-02</CLI_MSBuild_Version>
-    <CLI_NETSDK_Version>1.0.0-alpha-20161230-1</CLI_NETSDK_Version>
-    <CLI_WEBSDK_Version>1.0.0-alpha-20161205-1-154</CLI_WEBSDK_Version>
+    <CLI_NETSDK_Version>1.0.0-alpha-20170104-3</CLI_NETSDK_Version>
```
Comment (eerhardt) [inline build/Microsoft.DotNet.Cli.DependencyVersions.props]: Any chance you could bump this to `1.0.0-alpha-20170104-6` or later?  That way my fix for netcoreapp1.1 is included.

File: src/dotnet/commands/dotnet-new/FSharp_Console/$projectName$.fsproj [hunk 2]
```diff
@@ -4,6 +4,7 @@
     <OutputType>Exe</OutputType>
     <TargetFramework>netcoreapp1.0</TargetFramework>
     <Version>1.0.0-alpha</Version>
+    <EnableDefaultCompileItems>false</EnableDefaultCompileItems>
```
Comment (eerhardt) [inline src/dotnet/commands/dotnet-new/FSharp_Console/$projectName$.fsproj]: Should this go into the F# Sdk?

/cc @enricosada

Comment (livarcocc) [inline src/dotnet/commands/dotnet-new/FSharp_Console/$projectName$.fsproj]: Yes, I will file an issue on @enricosada repo for this. But this unblocks us.

**Comment by livarcocc** [comment]
@srivatsn

**Pull request merged**